helical/models/geneformer/geneformer_tokenizer.py

```diff
@@ -517,7 +517,7 @@ def tokenize_anndata(self, adata_file_path, target_sum=10_000):
                 n_counts = adata[idx].obs["n_counts"].values[:, None]
             else:
                 # If 'n_counts' doesn't exist, calculate it as the sum of counts for each cell
-                n_counts = np.sum(adata[idx, :].X, axis=1)[:, None]
+                n_counts = np.sum(adata[idx, :].X.toarray(), axis=1)[:, None]
```

Comment:
is there a way of doing that via a sparsearray api? I'm a little scared that this might blow up if we have a large sparse matrix since toarray() briongs everything into memory probably we would need to check the type..